### PR TITLE
VoyageAI embeddings integration

### PR DESCRIPTION
**Pull request by fzowl**

### CODE REVIEW

File: site/en/integrations/integrate_with_voyageai.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+Here we can find the main parameters that need to be modified for running with your own accounts. Beside each is a description of what it is.
+
+```python
+FILE = './content/books.csv'  # Download it from https://www.kaggle.com/datasets/jealousleopard/goodreadsbooks and save it in the folder that holds your script.
+COLLECTION_NAME = 'title_db'  # Collection name
+DIMENSION = 1024  # Embeddings size
+COUNT = 100  # How many titles to embed and insert.
+MILVUS_HOST = 'localhost'  # Milvus server URI
+MILVUS_PORT = '19530'
+MODEL_NAME = 'voyage-2'  # Which engine to use, please check https://docs.voyageai.com/docs/embeddings for available models
```
Comment (codingjaguar) [inline site/en/integrations/integrate_with_voyageai.md]: Nit, do you mean "Which model to use"?

File: site/en/integrations/integrate_with_voyageai.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+client = voyageai.Client(api_key="USE YOUR API KEY HERE")
```
Comment (codingjaguar) [inline site/en/integrations/integrate_with_voyageai.md]: what about `client = voyageai.Client(api_key="YOUR_VOYAGEAI_API_KEY")`?

File: site/en/integrations/integrate_with_voyageai.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+```
+
+This segment deals with Milvus and setting up the database for this use case. Within Milvus, we need to set up a collection and index the collection. For more information on how to use Milvus, look [here](https://milvus.io/docs/example_code.md).
```
Comment (codingjaguar) [inline site/en/integrations/integrate_with_voyageai.md]:
```suggestion
Then we need to connect to Milvus vector database to store and search the vector embeddings. Within Milvus, we need to create a collection and set up the index. For more information on how to use Milvus, look [here](https://milvus.io/docs/example_code.md).
```

File: site/en/integrations/integrate_with_voyageai.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+
+```python
+# Connect to Milvus
+connections.connect(host=MILVUS_HOST, port=MILVUS_PORT)
```
Comment (codingjaguar) [inline site/en/integrations/integrate_with_voyageai.md:55]: We have recently changed the API to a more friendly `MilvusClient` interface. Let's keep the code here as is, and I'll draft a follow up PR to fix the code.

File: site/en/integrations/integrate_with_voyageai.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+collection.create_index(field_name="embedding", index_params=index_params)
+```
+
+Once we have the collection setup we need to start inserting our data. This is in three steps: reading the data, embedding the titles, and inserting into Milvus.
+
+```python
+# Extract embedding from text using VoyageAI
+def embed(text):
+    response = client.embed(
+        texts=text,
```
Comment (codingjaguar) [inline site/en/integrations/integrate_with_voyageai.md]: maybe explain whether the input `text` is a string or a list of strings?

File: site/en/integrations/integrate_with_voyageai.md [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+        model=MODEL_NAME
+    )
+    return response.embeddings[0]
+
+
+# Insert each title and its embedding
+for idx, text in enumerate(random.sample(sorted(csv_load(FILE)), k=COUNT)):  # Load COUNT amount of random values from dataset
+    ins=[[idx], [(text[:198] + '..') if len(text) > 200 else text], [embed(text)]]  # Insert the title id, the title text, and the title embedding vector
```
Comment (codingjaguar) [inline site/en/integrations/integrate_with_voyageai.md]: may explain why using the magic numbers 198 and 200?

**Review by AnthonyTsu1984 — APPROVED** [review]

**Pull request merged**